src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -298,6 +298,14 @@ void GraphOptimizer::FuseConvMatmulFCDeconvAndDQScales(Graph& graph) {
                 return false;
             }
         }
+
+#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
+        // Per-channel DQ scales fusion is not supported by ACL
+        if (scalesDims[channelAxis] != 1) {
```

Comment:
Can we add an assertion with the same check somewhere in ACL executor? If the unsupported postops are unexpectedly fused, we need to throw an exception: this should save some time during debugging

Reply:
Added the assert into `ACLConvolutionExecutor` constructor.